docs/guides/doingatransformation.md

```diff
@@ -1,20 +1,30 @@
 # Doing a Transformation
 
-The Transformation object in TriggerMesh defines a set of operations that are
+The `Transformation` object in TriggerMesh defines a set of operations that are
 sequentially applied to incoming CloudEvents. In this guide, we will create a
-simple flow with an event producer and transformation to see the declarative syntax
-that is used for modifying events.
+simple flow (a.k.a Bridge) with an event producer and a transformation to see the declarative syntax that is used for modifying events.
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md). You can verify that the API is available with the following command:
+    
+    ```console
+       $ kubectl get crd|grep transformation
+       transformations.flow.triggermesh.io                     2021-10-06T09:01:40Z
+    ```
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Bumblebee](https://github.com/triggermesh/bumblebee)
+![](../assets/images/transformation.png)
+
+
+Let's create all the required objects:
+
+- [x] The sockeye target which serves as an event display.
+- [x] The `PingSource` which serves as an event producer.
+- [x] The `Transformation` to modify the produced events.
 
 ## Event display
 
-Before we begin to produce and transform events we need to have a tool to see
-results. Create a `sockeye` service:
+First of all, we need to have a tool to see filtering results. Create a `sockeye`
```

Comment:
Maybe replace "splitting results" with "events which the filter allowed to get through", or something along those lines?
Otherwise it sounds like the filter emits some "filtering results" of some sort (success, failure, ...).

Reply:
fixed